onnxruntime/core/dlpack/dlpack_converter.cc

```diff
@@ -229,13 +229,12 @@ OrtValue DlpackToOrtValue(DLManagedTensor* dlpack, bool is_bool_tensor) {
       dlpack->dl_tensor.data, info);
 
   OrtValue ort_value;
-  auto tensor_type = DataTypeImpl::GetType<Tensor>();
-  std::function<void(void*)> deleter = [dlpack, tensor_type](void* p) {
-    dlpack->deleter((dlpack));
-    tensor_type->GetDeleteFunc()(p);
+  std::function<void(void*)> deleter = [dlpack](void* p) {
+    dlpack->deleter(dlpack);
+    DataTypeImpl::GetType<Tensor>()->GetDeleteFunc()(p);
```

Comment:
if "DataTypeImpl::GetType<Tensor>()->GetDeleteFunc()(p)" is called, will be the data buffer released? then we got issue when calling " dlpack->deleter(dlpack);"? is this possible?

Reply:
Not possible. We use p_data to init Tenser without passing deleter (we don't have such deleter of type IAllocator*), it means the Tensor object will not take responsibility to release the p_data. I.e., p_data and p_tensor are independent to each other.

Reply:
I see. thanks!